src/sentry/deletions/base.py

```diff
@@ -177,15 +178,18 @@ def delete_instance(self, instance):
         try:
             instance.delete()
         finally:
-            self.logger.info(
-                'object.delete.executed',
-                extra={
-                    'object_id': instance_id,
-                    'transaction_id': self.transaction_id,
-                    'app_label': instance._meta.app_label,
-                    'model': type(instance).__name__,
-                }
-            )
+            # Don't log Group and Event child object deletions.
+            model_name = type(instance).__name__
+            if not re.search('^(Event|Group)(.+)', model_name):
```

Comment:
i would propose to use `re.compile` and to then use search on it. The re cache in Python is pretty stupid (if it hits max size it clears entirely).

Reply:
What does that look like in code form?

Reply:
Global variable like `_foo_re = re.compile(r'^(Event|Group)(.+)')` and then later `_foo_re.search(model_name)`.